Applications/Utils/FileConverter/NetCdfConverter.cpp

```diff
@@ -620,6 +622,12 @@ int main(int argc, char* argv[])
             "(http://www.opengeosys.org)",
         ' ', GitInfoLib::GitInfo::ogs_version);
 
+    TCLAP::ValueArg<int> arg_nodata(
+        "n", "nodata",
+        "explicitely specifies the no data value used in the dataset (usually it not necessary to set this)",
+        false, -9999, "integer specifying no data value");
```

Comment:
better to use the above constant in place of `-9999`, just in case it changes.

Reply:
✔️